build_all_ui_files.py

```diff
@@ -0,0 +1,46 @@
+BUILDER = r'C:\Users\OMi32458\Workspace\mantid\external\src\ThirdParty\lib\python2.7\pyuic4.bat'
```

Comment:
@EltayebAhmed. I added macros to mantid so that the cmake scripts automatically create the UI files. You might want to look into that to see how I got this working cross-platform. But as Martyn said `PyQt4.uic` is probably the right approach.